tests/e2e/run.sh

```diff
@@ -34,6 +34,7 @@ test_e2e() {
     "$ROOT/tests/complianceoperator/create.sh"
 
     deploy_stackrox
+    deploy_optional_e2e_components
```

Comment:
Why do we need this call here, in addition to where it's also added in `[tests/e2e/run-ui-e2e.sh](https://github.com/stackrox/stackrox/pull/6188/files#diff-3e55e891fe8143cfda664ff51ee9740a5d713e47ccfc555b2fbd9d865d9af8d7) ?

Reply:
I want this to be a general purpose post stackrox deploy method. Currently only used by UI by setting the ENV var but could be used for other e2es and other things to deploy.